README.md

```diff
@@ -31,12 +31,14 @@ Not required if pre-compiling the eBPF code (see [Installation options](#install
 ### Quickstart with Docker
 
 ```bash
-docker run --name tracee --rm --privileged --pid=host -v /lib/modules/:/lib/modules/:ro -v /usr/src:/usr/src:ro aquasec/tracee
+docker run --name tracee --rm --privileged --pid=host -v /lib/modules/:/lib/modules/:ro -v /usr/src:/usr/src:ro -v /tmp/tracee:/tmp/tracee aquasec/tracee
 ```
 
+> Note: You may need to change the volume mounts for the kernel headers based on your setup.
```

Comment:
Maybe add to this note something about KERN_SRC

Reply:
I added it in the detailed explanation below